.env_hooks/setup

```diff
@@ -13,7 +13,13 @@ fi
 
 
 if ! safe read secret/handshake &>/dev/null; then
-  cat > ${ENVIRONMENT_ROOT}/credentials.yml <<EOF
+  read -p "Vault is not working. Do you want to fix vault first? Otherwise you can generate credentails manully later. [yes or no]" YESNO
```

Comment:
"credentials" is spelled wrong.  Also, I'm not sure what you mean by "later"  It makes it sound like the script will continue, when it in fact aborts.